cli/lib/util.js

```diff
@@ -241,6 +241,14 @@ const parseOpts = (opts) => {
 
   debug('parsed cli options %o', cleanOpts)
 
+  if (cleanOpts.component && cleanOpts.e2e) {
```

Comment:
Can we move the error text to https://github.com/cypress-io/cypress/blob/develop/cli/lib/errors.js?

Reply:
Done.

I had not realized exactly how many different ways we have to throw errors in the command line. It might be a worthwhile project to examine and consider unifying at some point.